pkg/apis/v1alpha2/labels.go

```diff
@@ -91,9 +91,9 @@ var (
 	LabelSKUEncryptionAtHostSupported = Group + "/sku-encryptionathost-capable" // sku.EncryptionAtHostSupported
 
 	// GPU labels
```

Comment:
Consider mentioning on SKUName that it's equivalent to sku-name, but we offer it as the `sku` nomenclature is more familiar to Azure customers?